server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -231,14 +231,22 @@ func (s *BuildBuddyServer) CancelExecutions(ctx context.Context, req *inpb.Cance
 	return &inpb.CancelExecutionsResponse{}, nil
 }
 
-func makeGroups(ctx context.Context, groupRoles []*tables.GroupRole) []*grpb.Group {
-	r := make([]*grpb.Group, 0)
+func filterGroupRoles(ctx context.Context, groupRoles []*tables.GroupRole) []*tables.GroupRole {
```

Comment:
could we call this filterGroupsToSubdomain or something so it's clearer what it's filtering to?

(In general there's a lot of groups-specific code in this file now which maybe needs a better group-specific home...)

Reply:
done